dice-game-ui-angular/src/app/start-game/start-game.component.ts

```diff
@@ -26,7 +24,7 @@ export class StartGameComponent implements OnInit {
 
   startGame() {
     const newGame = new SimpleGameEngine(this.size, this.size);
-    this.gameStorageService.saveGame(newGame);
-    this.router.navigate(['/game']);
+    const gameGuid = this.gameStorageService.createGame(newGame);
+    this.router.navigate([`/game/${gameGuid}`]);
```

Comment:
Use array of all route parts values instead. `['/game', gameGuid]`;

Reply:
Changed

Reply:
Not changed.

Reply:
Sorry, didn't press a "push" button ))